File: mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("How tall should the pyramides be? ");
    }
    while(height < 1 || height > 8);

    int blanks = height - 1;
    int hashes = 1;

    // loop for the amount of rows
    for (int i = 0; i < height; i++)
    {
        for (int y = blanks; y != 0; y--)
        {
            printf(" ");
        }
        for (int z = hashes; z != 0; z--)
        {
            printf("#");
        }

        printf("  ");

        for (int z = hashes; z != 0; z--)
        {
            printf("#");
        }

        blanks--;
        hashes++;
        printf("\n");

    }
}

File: sentimental-mario-more/mario.py
def get_int(prompt):
    while(True):
        try:
            return int(input(prompt))
        except ValueError:
            pass

def main():

    height = get_int("Height: ")

    blanks = height - 1
    hashes = 1

    for _ in range(height):
        for blank in range(blanks):
            print(" ", end='')
        for hash in range(hashes):
            print("#", end ='')
        blanks -= 1
        hashes += 1
        print()

main()

